src/components/TopNav/SecondaryNav.tsx

```diff
@@ -1,25 +1,19 @@
 import { useState, useEffect } from "react";
-import klientoLogo from "../../assets/images/kliento-logo.png";
-import { secondaryNavLinks } from "./constants";
+import type { NavLink } from "./config";
 import { useNavigation } from "./hooks";
 import { MobileMenuToggle } from "./MobileMenuToggle";
 
 interface SecondaryNavProps {
-  currentService: string;
-  activeSection?: string | null;
+  serviceName: string;
+  serviceLinks: NavLink[];
+  logoSrc: string;
 }
 
-export function SecondaryNav({ currentService, activeSection }: SecondaryNavProps) {
+export function SecondaryNav({ serviceName, serviceLinks, logoSrc }: SecondaryNavProps) {
```

Comment:
This could be further simplified if instead of taking the `serviceName` and `serviceLinks` separately, you take the `serviceNavigation: NavLink`:

```ts
interface SecondaryNavProps {
  nav: NavLink;
  logoSrc: string;
}

export function SecondaryNav({ { href, text, children }: nav, logoSrc }: SecondaryNavProps) {
```